react/components/Charts/LineChart/index.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FC } from 'react'
+import { zipWith } from 'ramda'
```

Comment:
I would not import ramda just for one function. The project uses lodash now.

Reply:
Oh, okay. i will update